Forcepoint/forcepoint-swg/ingest/parser.yml

```diff
@@ -89,73 +89,73 @@ stages:
-          "1553": ["connection", "denied"]
-          "1554": ["connection", "denied"]
-          "1555": ["connection", "denied"]
-          "1556": ["connection", "allowed"]
-          "1557": ["connection", "denied"]
+          "1024": ["connection allowed"]
+          "1025": ["connection denied"]
+          "1026": ["connection allowed"]
+          "1027": ["connection denied"]
+          "1028": ["connection allowed"]
+          "1029": ["connection denied"]
+          "1030": ["connection allowed"]
+          "1031": ["connection denied"]
+          "1032": ["connection denied"]
+          "1033": ["connection denied"]
+          "1034": ["connection allowed"]
+          "1035": ["connection denied"]
+          "1037": ["connection allowed"]
+          "1040": ["connection allowed"]
+          "1041": ["connection denied"]
+          "1042": ["connection allowed"]
+          "1043": ["connection allowed"]
+          "1044": ["connection allowed"]
+          "1045": ["connection denied"]
+          "1046": ["connection denied"]
+          "1047": ["connection denied"]
+          "1048": ["connection allowed"]
+          "1049": ["connection denied"]
+          "1050": ["connection allowed"]
+          "1051": ["connection allowed"]
+          "1052": ["connection allowed"]
+          "1053": ["connection denied"]
+          "1054": ["connection denied"]
+          "1055": ["connection denied"]
+          "1056": ["connection allowed"]
+          "1057": ["connection denied"]
+          "1058": ["connection denied"]
+          "1059": ["connection denied"]
+          "1060": ["connection allowed"]
+          "1061": ["connection denied"]
+          "1062": ["connection allowed"]
+          "1063": ["connection allowed"]
+          "1064": ["connection denied"]
+          "1065": ["connection allowed"]
+          "1066": ["connection denied"]
+          "1067": ["connection allowed"]
+          "1068": ["connection denied"]
+          "1069": ["connection allowed"]
+          "1281": ["connection denied"]
+          "1282": ["connection allowed"]
+          "1293": ["connection allowed"]
+          "1296": ["connection allowed"]
+          "1299": ["connection allowed"]
+          "1301": ["connection denied"]
+          "1302": ["connection denied"]
+          "1303": ["connection denied"]
+          "1304": ["connection allowed"]
+          "1310": ["connection denied"]
+          "1311": ["connection denied"]
+          "1313": ["connection denied"]
+          "1314": ["connection denied"]
+          "1315": ["connection denied"]
+          "1316": ["connection allowed"]
+          "1317": ["connection denied"]
+          "1537": ["connection allowed"]
+          "1538": ["connection allowed"]
+          "1539": ["connection allowed"]
+          "1553": ["connection denied"]
+          "1554": ["connection denied"]
+          "1555": ["connection denied"]
+          "1556": ["connection allowed"]
+          "1557": ["connection denied"]
```

Comment:
ECS `event.type` field expected [severals of this allowed values](https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-type.html).
```suggestion
          "1024": ["allowed"]
          "1025": ["denied"]
          "1026": ["allowed"]
          "1027": ["denied"]
          "1028": ["allowed"]
          "1029": ["denied"]
          "1030": ["allowed"]
          "1031": ["denied"]
          "1032": ["denied"]
          "1033": ["denied"]
          "1034": ["allowed"]
          "1035": ["denied"]
          "1037": ["allowed"]
          "1040": ["allowed"]
          "1041": ["denied"]
          "1042": ["allowed"]
          "1043": ["allowed"]
          "1044": ["allowed"]
          "1045": ["denied"]
          "1046": ["denied"]
          "1047": ["denied"]
          "1048": ["allowed"]
          "1049": ["denied"]
          "1050": ["allowed"]
          "1051": ["allowed"]
          "1052": ["allowed"]
          "1053": ["denied"]
          "1054": ["denied"]
          "1055": ["denied"]
          "1056": ["allowed"]
          "1057": ["denied"]
          "1058": ["denied"]
          "1059": ["denied"]
          "1060": ["allowed"]
          "1061": ["denied"]
          "1062": ["allowed"]
          "1063": ["allowed"]
          "1064": ["denied"]
          "1065": ["allowed"]
          "1066": ["denied"]
          "1067": ["allowed"]
          "1068": ["denied"]
          "1069": ["allowed"]
          "1281": ["denied"]
          "1282": ["allowed"]
          "1293": ["allowed"]
          "1296": ["allowed"]
          "1299": ["allowed"]
          "1301": ["denied"]
          "1302": ["denied"]
          "1303": ["denied"]
          "1304": ["allowed"]
          "1310": ["denied"]
          "1311": ["denied"]
          "1313": ["denied"]
          "1314": ["denied"]
          "1315": ["denied"]
          "1316": ["allowed"]
          "1317": ["denied"]
          "1537": ["allowed"]
          "1538": ["allowed"]
          "1539": ["allowed"]
          "1553": ["denied"]
          "1554": ["denied"]
          "1555": ["denied"]
          "1556": ["allowed"]
          "1557": ["denied"]
```